packages/mgt-components/src/components/mgt-picker/mgt-picker.ts

```diff
@@ -286,6 +286,7 @@ export class MgtPicker extends MgtTemplatedComponent {
       if (parent) {
         parent.addEventListener('dataChange', (e: CustomEvent<DataChangedDetail>): void => this.handleDataChange(e));
       } else {
+        // eslint-disable-next-line @typescript-eslint/no-unsafe-call
```

Comment:
This one should not still be here.

Looks like there are a few others too, I'll clean them up so we can close out this work item